paper/basic_training.tex

```diff
@@ -1211,10 +1197,51 @@ \subsubsection{Grid based Ewald summation}
 
 
 \end{Checklists*}
+\todo[inline, color={yellow!20}]{DLM: Also need to point out the checklist above and discuss it in the text somewhere.}
+\todo[inline, color={yellow!20}]{DLM: I also need to go over the checklist again and make sure it is what we want/addresses key issues (and everything there is addressed in the text.}
+
+\section{Should you run MD?}
+
+A critical question \emph{before} preparing an MD simulation of your system is whether you even \emph{should} use MD for your system in view of the resources you have and what information you hope to obtain.
+MD is a tool, but it may not be the right tool for your problem.
+Before beginning any study, it is critical to sort out what questions you want to answer, what resources (computational and otherwise) you have at your disposal, and whether you have any information about your system(s) of interest that indicate you can realistically expect to answer those questions given a set of MD simulations.
+
+As noted above, the frequency of the fastest vibrational motions in a system of interest sets a fundamental limit on the timestep which, given fixed computational resources, sets a limit on how much simulation time can be covered with any reasonable amount of computer time.
+Thus, as noted in the Introduction, the longest all-atom MD simulations are on the microsecond to millisecond timescale.
```

Comment:
Should tag introduction (\ref{sec:Introduction}) if we get the chance.

Reply:
Done